modules/serverless-install-web-console.adoc

```diff
@@ -15,17 +15,13 @@ This procedure describes how to install and subscribe to the {ServerlessOperator
 image::serverless-search.png[{ServerlessOperatorName} in the {product-title} web console]
 
 . Review the information about the Operator and click *Install*.
+. On the *Install Operator* page:
 +
-image::serverless-operator.png[{ServerlessOperatorName} information]
-
-. On the *Create Operator Subscription* page:
-+
-image::serverless-create-sub.png[Create Operator Subscription page]
+image::serverless-create-sub.png[Install Operator page]
```

Comment:
Hmm. Isn't the image removed? I can see it when I scroll up in this PR. It's a little hard to assess when the preview is not available.

Reply:
Nope, just replaced. Only images/serverless-operator.png was deleted. If you look at the rich diff you can see how it changed :wink: